.github/PULL_REQUEST_TEMPLATE/version_bump_pr_template.md

```diff
@@ -0,0 +1,13 @@
+### Acceptance Criteria
+- Include here all things that this PR should solve
+
+### Checklist
+- [ ] Make sure you updated the `CURRENT_PROJECT_VERSION` with the appropriate release-candidate version in `ios/HathorMobile.xcodeproj/project.pbxproj`
+- [ ] Make sure you updated the `MARKETING_VERSION` with the appropriate version in `ios/HathorMobile.xcodeproj/project.pbxproj`
+- [ ] Make sure you updated the version attribute in `package.json`
+- [ ] Make sure you updated the version attribute in `package-lock.json`
+- [ ] Make sure you incremented the `versionCode` attribute in `android/app/build.gradle`
+- [ ] Make sure you updated the `versionName` with the appropriate version, including the release candidate number in `android/app/build.gradle`
+
+### Security Checklist
+- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.
```

Comment:
thought: Maybe this is unnecessary in this template. What do you think?

Reply:
I agree, done in 259bf8a64a533bedbfafbfc4033a4d42a4eebe70